data/data/gcp/cluster/master/main.tf

```diff
@@ -85,6 +85,7 @@ resource "google_compute_instance" "master" {
   metadata = {
     user-data = var.ignition
   }
+  metadata_startup_script = "echo \"Custom-DNS Test\"; echo \"Added by OpenShift\" >> /tmp/temp.conf; echo \"nameserver $(ip route get 8.8.8.8 | grep -oP 'src \\K[^ ]+')\" >> /tmp/temp.conf; cat /etc/resolv.conf >> /tmp/temp.conf; cat /tmp/temp.conf; sudo cp /tmp/temp.conf /etc/resolv.conf"
```

Comment:
Do we use any startup scripts on control-plane nodes? If so this is going to overwrite it. 

Also, do we need the custom dns test here? This is being applied no matter what configuration is chosen, so the print statements may not be appropriate.

Reply:
> Do we use any startup scripts on control-plane nodes? If so this is going to overwrite it.

Agreed. If we do, I was expecting it to be set in this file. I didn't see any so assuming we don't. Is there any other way to find out?

Added the conditional now.